packages/kbn-build/src/utils/npm.js

```diff
@@ -0,0 +1,49 @@
+import path from 'path';
```

Comment:
__nit:__ `path` and `writePkg` are not used (and probably we can get rid of `write-pkg` dependency).